### ISSUE TITLE
battle for undercity horde bug

### ISSUE THREAD

**Issue by sinhero** [dalaranwow/dalaran-wow#5236]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
thrall being bugged in random part of city while event isnt done. 
**Expected behaviour**: Tell us what should happen instead
not bugged 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by sinhero** [comment]
![wowscrnshot_010618_214619](https://user-images.githubusercontent.com/35185795/34647050-4c0b6094-f32c-11e7-9920-a599272ec923.jpg)
![wowscrnshot_010618_214619](https://user-images.githubusercontent.com/35185795/34647052-4ff90512-f32c-11e7-9273-726e60229554.jpg)

**Comment by TeridanDW** [comment]
Hello @sinhero,
Thrall is reseted in-game.
